templates/task/cf.yml

```diff
@@ -18,7 +18,7 @@ Parameters:
   ExecutionRole:
     Type: String
   Command:
-    Type: String
+    Type: CommaDelimitedList
```

Comment:
Does this require a documentation change? Like, is it backwards compatible if people just want to use double quotes?

Reply:
it's backwards compatible. it's not a new change but just a bug fix i think.